products/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from categories.models import Category_Join
+
+class Product(models.Model):
+    category_join = models.ForeignKey(Category_Join, on_delete=models.CASCADE)
+    name          = models.CharField(max_length=50)
+    product_group = models.ForeignKey('Product_Group', on_delete=models.CASCADE)
+    price         = models.DecimalField(decimal_places=2, max_digits=6)
+    ml            = models.IntegerField()
+    description   = models.CharField(max_length=500)
+    inventory     = models.IntegerField()
+
+    class Meta:
+        db_table = 'products'
+
+class Image(models.Model):
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
+    img_url = models.URLField()
```

Comment:
max_length 옵션을 넣어주세요. URLField의 경우 default로 200까지 밖에 설정되지 않습니다.
```suggestion
    img_url = models.URLField(max_legnth=1000)
```